_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -41,7 +41,7 @@ None of the target elements are part of [sequential focus navigation](https://ht
 
 ## Assumptions
 
-_There are currently no assumptions_
+No [user style sheets](https://drafts.csswg.org/css-cascade/#cascade-origin-user) are used and no changes to the [user agent default style sheet](https://drafts.csswg.org/css-cascade/#cascade-origin-ua) are in place, otherwise the test cases might have different outcomes of the ones presented here. 
```

Comment:
```suggestion
No [user style sheets](https://drafts.csswg.org/css-cascade/#cascade-origin-user) are used and no changes to the [user agent default style sheet](https://drafts.csswg.org/css-cascade/#cascade-origin-ua) are made. Otherwise the test cases might have different outcomes of the ones presented here. 
```

Wouldn't that sound better?

Reply:
I do prefer "are in place" over "are made". Are in place sounds more like a state (i.e. the user agent is in a specific state) while "are made" sounds more like an action (i.e. change the user agent to be in a specific state).

But this is not a strong preference, so if more people prefers the change I'll implement it.

Reply:
I am a bit uncomfortable having this as an **assumption**. It is not an assumption made by the rules, but one made by the examples. The rules work fine in presence of user agent styles.

I think it would be better maybe under the "Examples" heading (and before the "Passed Examples" one) as a note that the **examples** assume default styles.

Maybe also add a link to [Rendering](https://html.spec.whatwg.org/multipage/rendering.html#rendering) which is more or less what we consider to be the "default style".